src/renderers/common/IndirectStorageBufferAttribute.js

```diff
@@ -0,0 +1,17 @@
+import { BufferAttribute } from '../../core/BufferAttribute.js';
+
+class IndirectStorageBufferAttribute extends BufferAttribute {
```

Comment:
This should not be extended from `StorageBufferAttribute`? I think that way we wouldn't need so many checks.

Reply:
We could write indirect = false as standard in the constructor of StorageBufferAttribute and if we pass true, isIndirectStorageBufferAttribute would be triggered. This would allow us to forgo IndirectStorageBufferAttribute.

Reply:
I think extending the class is better for now.

Reply:
I take care of it tomorrow. Now I urgently need to go to bed. Thank you for taking the time to look at this

Reply:
No rush, take your time, thanks for your work :)